bigtable/doc/bigtable-hello-world.md

```diff
@@ -0,0 +1,90 @@
+# Example: C++ Hello World Application
+
+This example is a very simple "hello world" application, written in C++, that
+illustrates how to:
+
+* Connect to a Cloud Bigtable instance.
+* Create a new table.
+* Write data to the table.
+* Read the data back.
+* Delete the table.
+
+## Running the example
+
+This example uses the
+[Cloud Bigtable C++ Client Library](https://GoogleCloudPlatform.github.io/) to
+communicate with Cloud Bigtable.
+
+To run the example program, follow the
+[instructions](https://github.com/GoogleCloudPlatform/google-cloud-cpp/tree/master/bigtable/examples/)
+for the example on GitHub.
+
+### Including the Necessary Headers
+
+The example uses the following headers:
+
+@snippet bigtable_hello_world.cc bigtable includes
```

Comment:
This doesn't render anything special; I presume this is not a GitHub-supported tag.

Reply:
These are Doxygen tags, they render correctly in:

https://coryan.github.io/google-cloud-cpp/md_doc_bigtable-hello-world.html

Reply:
Ah, sorry, missed that in your PR description.

In that case... would it make sense to add a comment to the Markdown file that says this is not the place to view it, and link them to the other place? Thought that means the Doxygen file will also include the same thing, unless we add tags to strip it out...

Would you consider adding something like:

```html
<!--
    Note: the `@snippet`, `@see`, etc. tags used below are not processed by GitHub, but they
    will be expanded by Doxygen, and will be properly rendered on the official docs site:
    https://GoogleCloudPlatform.github.io/google-cloud-cpp/md_doc_bigtable-hello-world.html
-->
```

This will be invisible in Markdown and HTML, but will be visible to folks editing the Markdown file, so they're not tempted to remove it.

Thoughts?

Reply:
What about using `.dox` for Doxygen files, and not confusing ourselves with `.md` extensions for things that are only rendered correctly once Doxygen went through them?